source/infrastructure/data_connectors/orchestration/stepfunctions/workflow_orchestrator.py

```diff
@@ -198,42 +211,16 @@ def cdk_nag_suppression(self):
                 {
                     "id": 'AwsSolutions-IAM5',
                     "reason": nag_suppresion_reason,
-                    "appliesTo": ['Resource::*']
-                },
-                {
-                    "id": 'AwsSolutions-IAM5',
-                    "reason": nag_suppresion_reason,
-                    "appliesTo": ['Resource::<WorkflowOrchestrationAsyncCallbackConstructBrewRunJob52812FF2.Arn>:*']
-                },
-                {
-                    "id": 'AwsSolutions-IAM5',
-                    "reason": nag_suppresion_reason,
-                    "appliesTo": ["Resource::<HeadlessTransform07473135.Arn>:*"]
-                },
-                {
-                    "id": 'AwsSolutions-IAM5',
-                    "reason": nag_suppresion_reason,
-                    "appliesTo": ["Resource::<WorkflowOrchestrationBrewRunJob463FED05.Arn>:*"]
-                },
-                {
-                    "id": 'AwsSolutions-IAM5',
-                    "reason": nag_suppresion_reason,
-                    "appliesTo": ["Resource::<ConnectorUpdateFunction80A21979.Arn>:*"]
-                },
-                {
-                    "id": 'AwsSolutions-IAM5',
-                    "reason": nag_suppresion_reason,
-                    "appliesTo": ["Resource::<SalesforceWorkflowOrchestrationBrewRunJobC2FF95FB.Arn>:*"]
-                },
-                {
-                    "id": 'AwsSolutions-IAM5',
-                    "reason": nag_suppresion_reason,
-                    "appliesTo": ["Resource::<SalesforceWorkflowWorkflowOrchestrationBrewRunJob55408E3A.Arn>:*"]
-                },
-                {
-                    "id": 'AwsSolutions-IAM5',
-                    "reason": nag_suppresion_reason,
-                    "appliesTo": ["Resource::<WorkflowOrchestratorWorkflowOrchestrationBrewRunJob4557B9A3.Arn>:*"]
+                    "appliesTo": [
+                        'Resource::*',
```

Comment:
Is 'Resource::*' required here and other places since we are also enumerating the resources used for the policies?

Reply:
Ignoring the `Resource::*` will give the following error: `[Error at /SalesforceMarketingCloudStack/SalesforceWorkflow/S3TriggerDataBrewRunner/Role/DefaultPolicy/Resource] AwsSolutions-IAM5[Resource::*]: The IAM entity contains wildcard permissions and does not have a cdk-nag rule suppression with evidence for those permission.`. But I can remove some of the enumerated resources.